### DISCUSSION TITLE
New-AWSPowerShellLambdaPackage and the -PowerShellSdkVersion parameter

### DISCUSSION THREAD

**Discussion by halciber** [aws/aws-lambda-dotnet#1761] [Q&A] [answered]
Hello Everyone,

I have a problem where I have several versions of the .NET Core SDKs installed on my development machine. They are .NET6, 7, and 8 preview. I'm attempting to build a Lambda package for a PowerShell script. The original package is in .Net 6, and given the Lambda function itself, it makes sense (for now) to keep it at version 6.

The cmdlet I'm using to build the package is New-AWSPowerShellLambdaPackage, and when I run it, I got an error: _You must install or update .Net to run this application_. It effectively told me to run .NET 8. To work around this, I attempted to use the cmdlet parameter -PowerShellSdkVersion, to force it to use .NET 6. However. this has also failed, giving me the same error mentioned above.

I am wondering if there is a way to determine what the MS NuGet package manager version was for this older version by looking at what is installed? Any ideas?

Thanks,
Mike

**Comment by bhoradc** [comment, marked as answer]
Hi @halciber, 

I see that similar query is discussed in https://github.com/aws/aws-lambda-dotnet/discussions/1766. Please let us know if you still need any guidance on this Discussion. 

Regards, 
Chaitanya
